src/Command/PublishCommand.php

```diff
@@ -49,6 +51,13 @@ protected function configure()
                 InputOption::VALUE_OPTIONAL,
                 'Set it to "security" if something special is needed',
                 'bugfix'
+            )
+            ->addOption(
+                'elts',
+                null,
+                InputOption::VALUE_OPTIONAL,
```

Comment:
I guess we don't need a value at all for this option then `--elts` should be sufficient. (minor nitpick, but no fixing needed)

Reply:
I always fail at those argument / option things. I'll check this again.